guiEditor/src/components/propertyTab/propertyTabComponent.tsx

```diff
@@ -26,14 +29,22 @@ interface IPropertyTabComponentState {
 
 export class PropertyTabComponent extends React.Component<IPropertyTabComponentProps, IPropertyTabComponentState> {
     private _onBuiltObserver: Nullable<Observer<void>>;
-
+    private _timerIntervalId: number;
+    private _lockObject = new LockObject();
     constructor(props: IPropertyTabComponentProps) {
         super(props);
 
         this.state = { currentNode: null};
     }
 
+    timerRefresh() {
+        if (!this._lockObject.lock) {
+            this.forceUpdate();
+        }
+    }
+
     componentDidMount() {
+        this._timerIntervalId = window.setInterval(() => this.timerRefresh(), 500);

```

Comment:
If i remember correctly, this was done somewhere else as well. The question is, can it be prevented here? Is there no callback to check that it had changed, instead of checking every 500 ms?

Reply:
Discussed online :)